packaging/docker/README.md

```diff
@@ -4,9 +4,19 @@
 >
 > Also, the `latest` is now based on alpine, so **`alpine` is not updated any more** and `armv7hf` is now replaced with `armhf` (to comply with <https://github.com/multiarch> naming), so **`armv7hf` is not updated** either.
 
+## Prerequisites
+
+-   [Docker](https://docs.docker.com/install/#server)
+-   [Docker Compose](https://docs.docker.com/compose/install/)
+-   Domain configured in DNS pointing to host.
+
 ## Limitations
 
-Running Netdata in a container for monitoring the whole host, can limit its capabilities. Some data is not accessible or not as detailed as when running Netdata on the host.
+Running Netdata in a container is good for an internal network or to quickly analyze a host. 
+
+For monitoring the whole host, this can limit its capabilities. 
+Some data like the host OS performance or status is not accessible or not as detailed as when running Netdata on the host.
+A way around this is to provide special mounts to the Docker container so that Netdata can get visibility on host OS information like `/sys` and `/proc` folders or even `/etc/group` and shadow files.
 
 ## Package scrambling in runtime (x86_64 only)
```

Comment:
Placing that section there means it won't just be **Limitations**,  maybe "Limitations of running Netdata in Docker"

Reply:
Yes, we should change the limitations heading to **Limitations of running Netdata in Docker** regardless of what we do with the package scrambling section. Good point.

Reply:
>For increased security you can enable rescrambling of packages during runtime. To do this >set environment variable `RESCRAMBLE=true` while starting Netdata docker container.

The package scrambling section addresses a security concern, I think it can remain there.

Having this **Limitations of running Netdata in Docker** and then starting the section with "Running Netdata in a container works.." 🤔

Reply:
Perhaps we can make the package scrambling heading an h3 `###` to nest it beneath the limitations heading. I just want to de-prioritize its significance a bit in favor of getting straight to the Docker commands.

Reply:
Yes, I agree with this as well.

Reply:
RE: your last point, why don't we pull the paragraph about pros above the **Limitation...** heading? The beginning of the doc might look something like this (note the revised text beneath the **Package scrambling** heading, too, which we should incorporate):

```
Running Netdata in a container works great for an internal network or to quickly analyze a host. Docker helps you get set up quickly, and doesn't install anything permanent on the system, which makes uninstalling Netdata easy.

## Limitations of running Netdata in Docker

For monitoring the whole host, running Netdata in a container can limit its capabilities. 
Some data, like the host OS performance or status, is not accessible or not as detailed in a container as when running Netdata directly on the host.

A way around this is to provide special mounts to the Docker container so that Netdata can get visibility on host OS information like `/sys` and `/proc` folders or even `/etc/group` and shadow files.

### Package scrambling in runtime (x86_64 only)

Our x86_64 Docker images use [Polymorphic Polyverse Linux package scrambling](https://polyverse.io/how-it-works/). For increased security, you can enable rescrambling of Netdata packages during runtime by setting the environment variable `RESCRAMBLE=true` while starting Netdata with a Docker container.

## Run Netdata with the docker command
...
```

Reply:
>Running Netdata in a container works great for an internal network or to quickly analyze a host. 

It is not a limitation, so it should be above.